Arduino/README.md

```diff
@@ -0,0 +1,56 @@
+# 1. Micro-ros run_servo_service node
```

Comment:
Would be nice to create a script that does these steps so we don't need to do them manually every time

Reply:
resolved